saleor/graphql/shop/schema.py

```diff
@@ -27,7 +29,10 @@ class ShopQueries(graphene.ObjectType):
     )
     order_settings = PermissionsField(
         OrderSettings,
-        description="Order related settings from site settings.",
+        description=(
+            "Order related settings from site settings."
+            f"{DEPRECATED_IN_3X_INPUT} Use `channelUpdate` instead."
```

Comment:
The type fields have own field for marking it as a depreceated: `deprecation_reason`.

Reply:
We should also add there the info, that returned `orderSettings` will be for a first channel in alphabetical order.